desktop/src/com/unciv/app/desktop/ConsoleLauncher.kt

```diff
@@ -67,7 +67,7 @@ internal object ConsoleLauncher {
     private fun getGameParameters(civilization1: String, civilization2: String): GameParameters {
         return GameParameters().apply {
             difficulty = "Chieftain"
-            gameSpeed = GameSpeed.Quick
+            gameSpeed = "Quick"
```

Comment:
Good idea, I went ahead with those added those. Since I saw quick and standard both used in code I added const strings ```GameSpeed.DEFAULT``` and ```GameSpeed.DEFAULTFORSIMULATION```